core/chains/evm/txmgr/txmgr_test.go

```diff
@@ -563,9 +565,10 @@ func TestTxm_Reset(t *testing.T) {
 
 	txm := makeTestEvmTxm(t, db, ethClient, cfg, kst.Eth(), eventBroadcaster)
 
-	// 1 unconfirmed on each addr
-	cltest.MustInsertUnconfirmedEthTxWithBroadcastLegacyAttempt(t, borm, 4, addr)
 	cltest.MustInsertUnconfirmedEthTxWithBroadcastLegacyAttempt(t, borm, 2, addr2)
+	for i := 0; i < 1000; i++ {
+		cltest.MustInsertUnconfirmedEthTxWithBroadcastLegacyAttempt(t, borm, 4+int64(i), addr)
+	}
```

Comment:
Just one thousand was enough to cause this particular issue. The test is still only around 1s on my machine.